src/App.jsx

```diff
@@ -14,57 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [activeIndex, setActiveIndex] = useState(8);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const hendleClick = index => {
+    setActiveIndex(prevIndex => (prevIndex === index ? null : index));
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {activeIndex !== null ? (
+        <h1 className="title is-flex is-align-items-center">
+          {goods[activeIndex]} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setActiveIndex(null)}
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
+      <table className="table">
+        <tbody>
+          {goods.map((good, index) => {
+            return (
+              <tr
+                data-cy="Good"
+                className={
+                  activeIndex === index ? 'has-background-success-light' : ''
+                }
+              >
+                <td>
+                  <button
+                    data-cy={
+                      activeIndex === index ? 'RemoveButton' : 'AddButton'
+                    }
+                    type="button"
+                    className={
+                      activeIndex === index ? 'button is-info' : 'button'
+                    }
+                    onClick={() => hendleClick(index)}
```

Comment:
Ensure that the function `hendleClick` is renamed to `handleClick` to match the corrected function name.